tests/nnapi/specs/V1_2/log_softmax.mod.py

```diff
@@ -24,19 +24,19 @@ def test(input0, output0, input_data, beta, axis, output_data):
   }, model=model).AddVariations("relaxed", "float16")
 
 test(
-    input0=Input("input0", "TENSOR_FLOAT32", "{1, 1, 1, 2, 4}"),
-    output0=Output("output0", "TENSOR_FLOAT32", "{1, 1, 1, 2, 4}"),
+    input0=Input("input0", "TENSOR_FLOAT32", "{1, 1, 2, 4}"),
+    output0=Output("output0", "TENSOR_FLOAT32", "{1, 1, 2, 4}"),
```

Comment:
What is the reason to change 5-length into 4-length?? (just curious)

Reply:
I don'k know if we can edit the android's nnapi spec test script. -I used to generate new one postfix `_nnfw` if it doesn't run... @hseok-oh @glistening

Reply:
Rank 5 or higher is not supported, so it was modified to 4.

Reply:
@YiHyunJin Please don't modify nnapi original source, if you need to modify, please create another file based on original file with `_nnfw` in name and copyright. You can find similar examples by searching `*_nnfw.mod.py`.